openwisp_controller/config/handlers.py

```
@@ -152,3 +152,23 @@ def organization_disabled_handler(instance, **kwargs):
         # No change in is_active
         return
     tasks.invalidate_controller_views_cache.delay(str(instance.id))
+
+
+def organization_config_settings_change_handler(instance, **kwargs):
```

Ruff (0.14.10)

Unused function argument: `kwargs`

(ARG001)